contrib/externs/chrome_extensions.js

```diff
@@ -10262,3 +10262,116 @@ chrome.downloads.DeterminingFilenameEvent = function() {};
+
+
+/**
+ * @const
+ * @see https://developer.chrome.com/extensions/declarativeContent
+ */
+chrome.declarativeContent = {};
+
+
+/**
+ * @enum {string}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-PageStateMatcherInstanceType
+ */
+chrome.declarativeContent.PageStateMatcherInstanceType = {
+  DECLARATIVE_CONTENT_PAGE_STATE_MATCHER: ''
+};
+
+
+/**
+ * @enum {string}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-ShowPageActionInstanceType
+ */
+chrome.declarativeContent.ShowPageActionInstanceType = {
+  DECLARATIVE_CONTENT_SHOW_PAGE_ACTION: ''
+};
+
+
+/**
+ * @enum {string}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-SetIconInstanceType
+ */
+chrome.declarativeContent.SetIconInstanceType = {
+  DECLARATIVE_CONTENT_SET_ICON: ''
+};
+
+
+/**
+ * @enum {string}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-RequestContentScriptInstanceType
+ */
+chrome.declarativeContent.RequestContentScriptInstanceType = {
+  DECLARATIVE_CONTENT_REQUEST_CONTENT_SCRIPT: ''
+};
+
+
+/**
+ * @typedef {{
+ *   pageUrl: (!UrlFilter|undefined),
+ *   css: (!Array<string>|undefined),
+ *   isBookmarked: (boolean|undefined)
+ * }}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-PageStateMatcher
+ */
+chrome.declarativeContent.PageStateMatcherLiteral;
+
+
+/**
+ * @constructor
+ * @param {!chrome.declarativeContent.PageStateMatcherLiteral=} literalValue
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-PageStateMatcher
+ */
+chrome.declarativeContent.PageStateMatcher = function(literalValue) {};
+
+
+/**
+ * @constructor
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-ShowPageAction
+ */
+chrome.declarativeContent.ShowPageAction = function() {};
+
+
+/**
+ * @typedef {{
+ *   imageData: (!ImageData|!Object<number, !ImageData>|undefined)
+ * }}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-SetIcon
+ */
+chrome.declarativeContent.SetIconLiteral;
+
+
+/**
+ * @constructor
+ * @param {!chrome.declarativeContent.SetIconLiteral=} literalValue
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-SetIcon
+ */
+chrome.declarativeContent.SetIcon = function(literalValue) {};
+
+
+/**
+ * @typedef {{
+ *   css: (!Array<string>|undefined),
+ *   js: (!Array<string>|undefined),
+ *   allFrames: (boolean|undefined),
+ *   matchAboutBlank: (boolean|undefined)
+ * }}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-RequestContentScript
+ */
+chrome.declarativeContent.RequestContentScriptLiteral;
+
+
+/**
+ * @constructor
+ * @param {!chrome.declarativeContent.RequestContentScriptLiteral=} literalValue
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-RequestContentScript
+ */
+chrome.declarativeContent.RequestContentScript = function(literalValue) {};
+
+
+/**
+ * @type {!ChromeEvent}
```

Comment:
Please use `ChromeVoidEvent`
We don't want to add any more uses of the loosely typed `ChromeEvent`.